app/Providers/SimpleSamlServiceProvider.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace App\Providers;
+
+require_once __DIR__ . '/../../simplesamlphp/lib/_autoload.php';
+
+use Illuminate\Support\ServiceProvider;
+use App\Helpers\SimpleSaml;
+
+class SimpleSamlServiceProvider extends ServiceProvider
+{
+    public const SAML_AUTH_ID = 'cornell';
+
+    /**
+     * Register services.
+     *
+     * @return void
+     */
+    public function register()
+    {
+        $auth          = new \SimpleSAML\Auth\Simple(self::SAML_AUTH_ID);
+        $simplesamlphp = new SimpleSaml($auth, self::SAML_AUTH_ID);
```

Comment:
SimpleSaml only takes the one argument...
```suggestion
        $simplesamlphp = new SimpleSaml($auth);
```